alerting/pagerduty/README.md

```diff
@@ -21,7 +21,7 @@ module "signalfx-integrations-alerting-pagerduty" {
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:-----:|
-| api\_key | Pagerduty API token | `string` | n/a | yes |
```

Comment:
this is autogenerated from variables descriptions so you need to update the corresponding variable description also or let it as it is

Reply:
variables.tf updated